### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by Osoro254Alex**
### I did the following in this milestone of the Capstone project:

- The project is a single-page application (SPA) built with React and Redux.
- Data retrieved from the API should be stored in the Redux store.
- Filter the data that you retrieve from the API using a Filter stateless component.
- Every page (the main page and the pages for each item) should have a unique route within the SPA.
- Project be deployed and accessible online (add a link to the online version of your app to your README file),
- Capstone Project [Live Demo Link](https://explore-the-world-countries.netlify.app/),
- Loom [video Link](https://loom.com/share/fe39b7534cec4cdd86d7c30ed409f5d9)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,226 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- Create a local directory that you want to clone the repository.
+
+- Open the command prompt in the created directory.
+
+- On the terminal run this command git clone https://github.com/Osoro254Alex/Bookstore
```
Comment (DammyShittu) [inline README.md]: - [ ] Your link for cloning the app is incorrect. Kindly fix that

File: README.md [hunk 2]
```diff
@@ -1,70 +1,226 @@
...
+
+```
+git clone https://github.com/Osoro254Alex/Bookstore.git
+```
+
+or
+
+```
+git clone git@github.com:Osoro254Alex/Bookstore.git
+```
```
Comment (DammyShittu) [inline README.md]: - [ ] Kindly fix your github links here as well

File: README.md [hunk 3]
```diff
@@ -1,70 +1,226 @@
...
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
```
Comment (DammyShittu) [inline README.md]: - [ ] Kindly include an acknowledgment for Nelson Sakwa according to the Creative Commons License requirement.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Osoro254Alex,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Osoro254Alex** [comment]
@DammyShittu 
- Update README.md file.

File: README.md [hunk 4]
```diff
@@ -1,70 +1,236 @@
...
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+- Thanks to the Microverse team for the great curriculum.
+- Hat tip to anyone whose code was used
+- A great thanks to My coding partners, morning session team, and standup team Partners for their contributions.
```
Comment (mirouhml) [inline README.md:223]: - [ ] As the previous reviewer mentioned you need to give the appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/), kindly do that.

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @Osoro254Alex, 

Great work on making the changes requested by the previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet, but don't worry you're almost there!! 😉

image

 ## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] Please know that your README should also have the section Future Features, including 1-3 potential features to be added.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Osoro254Alex** [comment]
@mirouhml 

- Read me file Update

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Osoro254Alex👋, 

### You did a marvelous job on the React capstone project and this is one of the best projects I have seen so far!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- Awesome UI design. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Osoro254Alex** [comment]
@DammyShittu  
Thanks for your review
@mirouhml  
Thanks for your review
@thecodechaser 
Thanks for your review

**Pull request merged**